src/compass/utils/lut.py

```diff
@@ -40,8 +40,8 @@ def cumulative_correction_luts(burst, dem_path,
                                           rg_step=rg_step,
```

Comment:
Sorry that  I can't directly comment on the line, but should we remove `scratch_path`? I'm getting an unused variable error from flake8.

Reply:
alternatively- did you mean to pass `scratch_path` as an argument to `compute_geocoding_correction_luts`?